doc/http_api.md

```diff
@@ -0,0 +1,108 @@
+# HTTP API
+
+Enables mods to make HTTP requests.
+
+## Settings
+
+To use this api, mods need to be listed in one or the other following 
+settings in csv format:
+* `secure.trusted_mods`
+* `secure.http_mods`
+
+An example would be:
+
+```
+secure.http_mods = modname1, modnametwo, modenamethree
+```
+
+## Types
+
+### HttpRequest
+
+A table consisting of the following paramaters:
+
+* url (required): 
```

Comment:
I would format any code value in backticks for clarity

```suggestion
* `url` (required): 
```

Reply:
https://github.com/minetest/minetest_docs/pull/56/commits/31405bcab27031ae5769c38af5f44d13a5beb7e7